.circleci/config.yml

```diff
@@ -8,7 +8,7 @@ orbs:
 executors:
   docker-python:
     docker:
-      - image: circleci/python:3.7
+      - image: cimg/python:3.13.2
```

Comment:
You can drop the patch version here. Keep it as 3.13 that way the latest patch will be applied each run w/o needing to edit the config.